CHANGELOG.md

```diff
@@ -7,6 +7,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- PanelSidebar cypress test and toggle it on any screen size
+- `collapseIconOnly` to `PanelItem` in order to collapse `PanelSideBarItem` only by clicking on chevron icon and label can work as link
+- PanelLayout
+
+### Merged
+
+- Merged `PaneMenulItem` into `PanelItem`
```

Comment:
ChangeLog should refer now to the main version of the package. I know that it's not simpler but can you try to summarize new features? Of course these change in them middle should not appear in the changelog

Reply:
@manni497 Changelog updated. Inputs are welcome :)

Reply:
Also few details added in README